src/tests/integration/mapping.test.ts

```diff
@@ -0,0 +1,38 @@
+import { startAppServer } from '../../server/server'
+import fetch from 'node-fetch'
+
+const TEST_PORT = process.env.PORT || 50604
+const ADMIN = process.env.ADMIN || 'hjhj'
+const apiURL = `http://127.0.0.1:${TEST_PORT}`
+
+describe('/api', () =>{
+  let server 
+
+  beforeAll(async ()=>{
+    server = await startAppServer(TEST_PORT, ADMIN)
+  })
+
+  afterAll(()=>{
+    server.close()
+  })
+
+  it('checks mappings for newly added mapping', async()=>{
+    await fetch(`${apiURL}/api/mappings`, {
+      method: 'POST', 
+      headers: {
+        authorization: ADMIN,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        domain: 'Rahul',
+        subDomain: 'testing',
```

Comment:
hmmm.... creating 2 same subDomains seems to be a problem, creating an issue to track this:
https://github.com/garageScript/myProxy/issues/244

For now, I think you need a variable to guarantee uniqueness for every time this test runs. 
```
const subDomain = `testing${uuidv4()}`
...
    body: JSON.stringify({
      domain...
      subDomain....
     })
....
```